packages/@tinacms/cli/src/cmds/start-server/server.ts

```diff
@@ -61,6 +61,39 @@ const gqlServer = async () => {
     return res.json(result)
   })
 
+  app.get('/list-branches', async (req, res) => {
+    try {
+      const { query } = req
+      const { owner, repo } = query
+      const result = await gqlPackage.listBranches({
+        auth: process.env.GITHUB_PERSONAL_ACCESS_TOKEN,
```

Comment:
won't this have to be proxied though `tina-cloud?

Reply:
Yes, I imagine we'll be taking steps to get this cleaned up and more in line with some best practices. This first pass is really, "Hey, can this work?"